app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
> My expectations are that I would be able to see the stack trace when I get an exception in development. This was not clear from the above?

I think so, that was just my standard 'reproductions and expectations' sentence.

This is the page you want to see right:
![sinatra error](https://cloud.githubusercontent.com/assets/5096681/26419346/b6185cdc-40bf-11e7-93d6-6ff912d94578.png)